src/apple/apple-wizard.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable max-lines */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as templates from './templates';
+import * as xcManager from './xcode-manager';
+import * as codeTools from './code-tools';
+import * as bash from '../utils/bash';
+import { WizardOptions } from '../utils/types';
+
+const xcode = require('xcode');
+/* eslint-enable @typescript-eslint/no-unused-vars */
+
+import {
+  askForProjectSelection,
+  askForSelfHosted,
+  askForWizardLogin,
+  askToInstallSentryCLI,
+  SentryProjectData,
+  printWelcome,
+  abort,
+} from '../utils/clack-utils';
+
+export async function runAppleWizard(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Apple Wizard',
+    promoCode: options.promoCode,
+  });
+
+
+  if (!bash.hasSentryCLI()) {
+    if (!(await askToInstallSentryCLI())) {
+      clack.log.warn("Without Sentry-cli, you won't be able to upload debug symbols to Sentry. You can install it later by following the instructions at https://docs.sentry.io/cli/");
+    } else {
+      await bash.installSentryCLI();
+    }
+  }
+
+  const projectDir = process.cwd();
+  const xcodeProjFile = findFilesWithExtension(projectDir, ".xcodeproj")[0];
+
+  if (!xcodeProjFile) {
+    clack.log.error('No xcode project found. Please run this command from the root of your project.');
+    await abort();
+    return;
+  }
+
+  const pbxproj = path.join(projectDir, xcodeProjFile, "project.pbxproj");
```

Comment:
`m`: Does this also work for `.xcworkspace`?

Reply:
That depends. `.xcworkspace` just groups projects in a workspace, it alone is irrelevant to the wizard. 
But if we have two or more `.xcodeproj` in the same directory, which may be common for someone using `.xcworkspace`, than we're changing the first one the wizard finds, we should ask which project should be updated.

The easy answer is that `.xcworkspace` does not matter for the wizard, but we should check whether there is more than one project in the same directory.

Reply:
Yes, please. Let's do that. Thanks for the clarification, @brustolin.